### ISSUE TITLE
instructions for new plugin creation result in connection failure

### ISSUE THREAD

**Issue by daniel-garcia** [turbot/steampipe#1112]
When following the instructions on https://steampipe.io/docs/develop/writing-plugins#writing-plugins , restarting the steampipe service results in an error where the plugin can't be located. It appears that steampipe is making an assumption that the plugins from hub.steampipe.io despite the fact that they are developed locally. Below is the trace output from restart. 

```
2021-11-13T11:20:19.423-0500 [TRACE] steampipe: plugin path /Users/dgarcia/.steampipe/plugins/hub.steampipe.io/plugins/local/promql@latest not found - searching for folder using hashed name
2021-11-13T11:20:19.423-0500 [TRACE] steampipe: close local client 0xc000318780
Error: failed to load connection 'promql': no plugin installed matching local/promql
/Users/dgarcia/.steampipe/config/promql.spc:1,1-11
```
The config file contents:
```
dgarcia@rm-ml-dgarcia ~ % cat .steampipe/config/promql.spc
connection "promql" {
    plugin    = "local/promql"
}
```

**Steampipe version (`steampipe -v`)**
Example: v0.9

**To reproduce**
Start with a new steampipe environment on OSX. Follow the instructions to create your own NEW plugin. 

**Expected behavior**
The plugin should load or instructions should be updated to reflect how to actually get the plugins to work.

**Comment by judell** [comment]
>  It appears that steampipe is making an assumption that the plugins from hub.steampipe.io 
> despite the fact that they are developed locally.

Yes, good point. This makes it possible for a beginning plugin writer to clone a plugin repo, change it slightly, run `make`, and see the results. When you create a whole new plugin in this way, by cloning an existing one, you can again just type `make` to install. 

So the needed copy command is to `~/.steampipe/plugins/hub.steampipe.io/plugins/turbot/YOUR_PLUGIN@latest`. 

Do we want to document it that way, or would we rather make the existing instructions work as written?

**Comment by judell** [comment]
FWIW I made myself a barebones plugin, https://github.com/judell/steampipe-plugin-hello, that I can clone and modify to  start on a new plugin project. Because it was cloned from a hub plugin it has all the same machinery, and same assumptions about deploying locally to `~/.steampipe/plugins/hub.steampipe.io/plugins/turbot/`. 

When I clone it to start a new plugin, there's a fair bit of renaming that needs to occur. I haven't automated that yet but of course it could be automated, so you'd have a startup experience like `rails new app_name` or similar.  Doing things this way would imply that every plugin aspires to join the hub, though, and I'm not sure that's a correct default, what do others think?

**Comment by kaidaguerre** [comment]
fixed by 73d568c
